controllers/topicPage/TopicPage.js

```diff
@@ -88,6 +96,9 @@ class TopicPage {
       const listBlockUser = await getListBlockUser(req.userId);
       const listBlockDomain = await getBlockDomain(req.userId);
       const listPostAnonymous = await getListBlockPostAnonymous(req.userId);
+      const myAnonymousUser = await UsersFunction.findAnonymousUserId(User, req.userId, {
+        raw: true
+      });
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Ensure that error handling is in place for the `UsersFunction.findAnonymousUserId` function call. If the function fails due to any reason (like database connectivity issues), it should not crash the entire application.
```diff
+    try {
+      const myAnonymousUser = await UsersFunction.findAnonymousUserId(User, req.userId, { raw: true });
+    } catch (error) {
+      // Handle error appropriately
+    }
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->